README.md

```
@@ -4,43 +4,108 @@ A cross-platform app using `drift` for local persistence.
 
-## Development
+1. Clone the repository:
 
-As this app uses drift, it depends on code-generation.
-Use `dart run build_runner build -d` to automatically build the generated
-code.
+```shell
+git clone https://github.com/tsinis/sealed_world.git && cd app
+```
+
+2. Install dependencies:
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)